### ISSUE TITLE
KapeFiles _BasicCollection target broken in repacked autoexec config

### ISSUE THREAD

**Issue by heck-gd** [Velocidex/velociraptor#476]
When repacking the following config into a binary, the collection ends after lazy_ntfs looked at $MFT, $LogFile and $Boot. The resulting archive is only around 2 KB in size. The log doesn't contain an error of any sort.

This is a v0.4.3 regression (it remains broken in v0.4.4 and v0.4.5). In v0.4.2 it collects everything one would expect and the resulting archive is close to 100 MB on my test system.

```
autoexec:
  argv: ["artifacts", "collect", "-v", "Windows.KapeFiles.Targets",
         "--output", "collection_$COMPUTERNAME.zip",
         "--args", "_BasicCollection=Y",
         "--args", "VSSAnalysis=Y"]
```

When replacing `_BasicCollection` with several individual targets, the issue does not occur.

**Comment by scudette** [comment]
This was a regression in 0.4.5 which is fixed in head. To use the latest head binary from the ci pipeline you need to also add it as a third party binary (so the offline collector builder can use it)

```
Velociraptor third_party upload --name VelociraptorWindows velociraptor.exe
```

We hope to make a new release (0.4.6) by next week.

**Comment by scudette** [comment]
Should be fixed with 0.4.6
